plugins/wallabag.koplugin/main.lua

```diff
@@ -47,11 +48,21 @@ function Wallabag:init()
     self.username = self.wb_settings.data.wallabag.username
     self.password = self.wb_settings.data.wallabag.password
     self.directory = self.wb_settings.data.wallabag.directory
-    self.is_delete_finished = self.wb_settings.data.wallabag.is_delete_finished
-    self.is_delete_read = self.wb_settings.data.wallabag.is_delete_read
-    self.is_auto_delete = self.wb_settings.data.wallabag.is_auto_delete
-    self.is_sync_remote_delete = self.wb_settings.data.wallabag.is_sync_remote_delete
-    self.filter_tag = self.wb_settings.data.wallabag.filter_tag
+    if self.wb_settings.data.wallabag.is_delete_finished ~= nil then
```

Comment:
I'm a little confused by this. Aren't you basically saying the same thing because `self.is_delete_finished` will still remain nil if the new value is nil?

Reply:
The self.is_* variables have default values set at the top of init(). If there is a config file to read from, then the defaults are replaced otherwise keep the default.

Reply:
Alright, thanks.